core/src/test/java/com/datastax/dse/driver/internal/core/cql/reactive/TestSubscriber.java

```diff
@@ -69,6 +80,10 @@ public Throwable getError() {
     return elements;
   }
 
+  public boolean isCompleted() {
```

Comment:
I need it because those two lines don't work for my test case
```
mainSubscriber.awaitTermination();
assertThat(mainSubscriber.getError()).isNull();
```

`mainSubscriber.awaitTermination()` does not throw any exception if after timeout there is still data in publisher.  In this regard, the question, is it correct behaviour that `mainSubscriber.awaitTermination()` does not throw exception after timeout ?

Reply:
> is it correct behaviour that mainSubscriber.awaitTermination() does not throw exception after timeout ?

Arguably not :-)
Feel free to change the method to throw throw an assertion error if the timeout fires.

Reply:
I deleted isCompleted method and fixed `awaitTermination`